subsys/mpsl/cx/bluetooth/mpsl_cx_1w_bluetooth.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ *   This file implements a generic bluetooth external radio 1w coexistence
+ *   interface.
+ */
+#include <stddef.h>
+#include <stdint.h>
+
+#include <device.h>
+#include <drivers/gpio.h>
+#include <devicetree.h>
+#include <nrfx.h>
+#include <nrfx_ppi.h>
+#include <nrfx_gpiote.h>
+#include <mpsl_coex.h>
+
+#define COEX_TIMER NRF_TIMER1
+
+#if DT_HAS_COMPAT_STATUS_OKAY(sdc_radio_coex_one_wire)
```

Comment:
we use a different way now, because upstream asked for it. 
You have to use a pointer in the radio node to the coex node, and resolve that pointer instead.
This is how do for 3-wire coex:
```c
#if DT_NODE_HAS_STATUS(DT_PHANDLE(DT_NODELABEL(radio), coex), okay)
#define COEX_NODE DT_PHANDLE(DT_NODELABEL(radio), coex)
#else
#define COEX_NODE DT_INVALID_NODE
#error No enabled coex nodes registered in DTS.
#endif

#if !(DT_NODE_HAS_COMPAT(COEX_NODE, sdc_radio_coex_three_wire))
#error Selected coex node is not compatible with sdc-radio-coex-three-wire.
#endif
```